scripts/launch/demo.sh

```diff
@@ -17,7 +17,7 @@ if [[ ! -d "$base/../../.stack-work" ]]; then
 fi
 
 # Define the default amount of nodes to run
-DEFAULT_NODES_N=3
+DEFAULT_NODES_N=4
```

Comment:
can you throw an error if `N /= 4` ? Just in case, because it doesn't work otherwise.

Reply:
What exactly doesn't work?